src/main/java/com/ft/methodecontentcollectionmapper/MethodeContentCollectionMapperApplication.java

```diff
@@ -57,7 +60,20 @@ public void run(final MethodeContentCollectionMapperConfiguration configuration,
         environment.jersey().register(new VersionResource());
         environment.jersey().register(new BuildInfoResource());
 
-        EomFileToContentCollectionMapper eomContentCollectionMapper = new EomFileToContentCollectionMapper();
+        DocumentStoreApiClient documentStoreApiClient = new DocumentStoreApiClient(
```

Comment:
Why not use the `DocumentStoreApiClient(UppServiceConfiguration uppServiceConfiguration, Environment environment)` constructor?